<launch>
  <!-- Launch the path, leader, and driver for navigation. -->
  <node pkg="drive_stack" type="path.py" name="path" output="screen">
    <param name="max_speed" value="2.0" type="double"/>
  </node>
  <node pkg="drive_stack" type="leader_nonlinear_force.py" name="leader" output="screen"/>
  <node pkg="drive_stack" type="driver_pseudolinear.py" name="driver" output="screen"/>
</launch>
